config/locales/models/activity.en.yml

```diff
@@ -277,8 +277,8 @@ en:
         policy_marker_nutrition: Nutrition Policy
         programme_status: Activity status
         projects: Projects (level C)
-        recipient_country: Recipient country
-        recipient_region: Recipient region
+        recipient_country: Recipient country (read only)
```

Comment:
Do we want to use `read only` here? It seems like an overly technical term. How about `Legacy field: not editable` or something? I'm also wondering if we can put it in smaller text - say wrapped in a span with a `govuk-body-s` class?

Reply:
Can do.